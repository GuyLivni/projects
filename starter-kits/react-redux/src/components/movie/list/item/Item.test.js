/* eslint-disable no-undef */
import renderer from "react-test-renderer";
import React from "react";
import MovieItem from "./";

describe("MovieItem", () => {
  it("MovieItem ui renders correctly", () => {
    const tree = renderer
      .create(
        <MovieItem
          name="batman"
          summary="batman test"
          image="http://static.tvmaze.com/uploads/images/medium_portrait/4/10842.jpg"
          rating={22}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
