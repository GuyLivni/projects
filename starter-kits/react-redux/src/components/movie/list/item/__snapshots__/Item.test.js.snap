// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieItem MovieItem ui renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border: 2px solid #1b1d2d;
  box-sizing: border-box;
  padding: 5px;
  margin: 5px;
  border-radius: 5px;
  cursor: pointer;
  background: #0b0f1f;
}

.c0:hover {
  border-color: #282b42;
}

.c0:active {
  -webkit-transform: scale(0.97);
  -ms-transform: scale(0.97);
  transform: scale(0.97);
}

.c2 {
  text-align: left;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-right: 10px;
  height: 35px;
}

.c3 {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  max-width: 330px;
  font-weight: 600;
  color: #ffffff;
}

.c4 {
  color: #9194a2;
}

<div
  className="c0"
  onClick={[Function]}
>
  <img
    alt="image"
    className="c1"
    src={undefined}
  />
  <div
    className="c2"
  >
    <div
      className="c3"
    />
    <div
      className="c4"
    />
  </div>
</div>
`;
